config/locales/en.yml

```diff
@@ -188,6 +188,21 @@ en:
         title: Site Privacy Policy
       site_title: Site title
       title: Site Settings
+    statuses:
+      back_to_account: Back to account page
+      batch:
+        delete: Delete
+        nsfw_off: NSFW OFF
+        nsfw_on: NSFW ON
```

Comment:
"Content warning" is a function to hide sentences, and NSFW is a function to hide images. The implemented function is a function to change NSFW.